external/implementations/samlconf-openam-impl/src/main/java/org/codice/compliance/saml/plugin/openam/OpenAMDataParser.java

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+package org.codice.compliance.saml.plugin.openam;
+
+import com.google.common.collect.ImmutableMap;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import org.codice.compliance.utils.TestCommon;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+class OpenAMDataParser {
+
+  //  A class only for retrieving values from the openAM data file.
+  private final ImmutableMap<String, String> formMapping;
+  private final String openAMRealm;
+  private final String baseURL;
+
+  private static final String REALM = "realmName";
+  private static final String BASE_URL = "baseUrl";
+  private static final String LOGIN_FORM = "loginForm";
+  private static final String INPUT = "input-";
+  private static final String OUTPUT = "output-";
+
+  private static final String USER_ARG = "USER.ARG";
+  private static final String PASSWORD_ARG = "PASS.ARG";
+
+  private static final String fileName = "OpenAMLoginData.xml";
+
+  OpenAMDataParser() {
+
+    DocumentBuilder doc;
+    try {
+      DocumentBuilderFactory docBF = DocumentBuilderFactory.newInstance();
+      doc = docBF.newDocumentBuilder();
+    } catch (ParserConfigurationException e) {
+      throw new IllegalStateException(e);
+    }
+
+    Document xml;
+    try {
+      xml = doc.parse(new File(TestCommon.Companion.getImplementationPath(), fileName));
```

Comment:
:blue_book: We originally agreed on having this be a Java Stream but the doc.parse method could not take a String as a correct parameter. Instead of finding a new class I just created a new File object and passed that in.